docs/website/docs/dlt-ecosystem/verified-sources/filesystem/advanced.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Advanced filesystem usage
+description: Use filesystem source as a building block
+keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+---
+
+The filesystem source actually provides you with the building blocks to facilitate loading data from files. This section aims to give you more information about how you can customize the filesystem source for your use case.
```

Comment:
The filesystem source provides the building blocks to load data from files. This section explains how you can customize the filesystem source for your use case.